nat-lab/tests/utils/vm/mac_vm_util.py

```diff
@@ -62,12 +63,10 @@ async def _copy_binaries(
                 raise exception
         await connection.create_process(["mkdir", "-p", directory]).execute()
 
-    DIST_PATH = "dist/darwin/macos/release/x86_64/"
+    DIST_PATH = f"dist/darwin/macos/{os.getenv('TELIO_BIN_PROFILE')}/x86_64/"
```

Comment:
How will this act if TELIO_BIN_PROFILE is missing?

Reply:
it raises an exception [here](https://github.com/NordSecurity/libtelio/pull/823/files#diff-a91e5f9ea38f2b6c0e059f4f4ad014acb21345cc7a344656df63eb5379d51afaR61) and asks user to define it.